app/routes/circulars/circulars.lib.ts

```diff
@@ -52,6 +52,7 @@ type SubjectMatcher = [RegExp, (match: RegExpMatchArray) => string]
 const subjectMatchers: SubjectMatcher[] = [
   [/GRB[.\s_-]*(\d{6}[a-z|.]\d*)/i, ([, id]) => `GRB ${id.toUpperCase()}`],
   [/SGR[.\s_-]*(J*\d{4}\.?\d*\+\d{4})/i, ([, id]) => `SGR ${id.toUpperCase()}`],
+  [/FRB[.\s_-]*(\d{8}[a-z|.]\d*)/i, ([, id]) => `FRB ${id.toUpperCase()}`],
```

Comment:
There is already an FRB subject matcher. It's the last matcher in this list. Why do we need a second one?